drivers/gles3/shaders/canvas.glsl

```diff
@@ -332,7 +330,7 @@ vec4 light_shadow_compute(uint light_base, vec4 light_color, vec4 shadow_uv
 		shadow /= 13.0;
 	}
 
-	vec4 shadow_color = unpackUnorm4x8(light_array[light_base].shadow_color);
+	vec4 shadow_color = godot_unpackUnorm4x8(light_array[light_base].shadow_color);
```

Comment:
Could likely keep using the `unpackUnorm4x8` name since you `#define`d it as alias to our `godot_` function.

Reply:
As discussed on chat, this was kept like this on purpose to make it clear we're using emulated functions.